api/data_explorer/test/test_dataset_controller.py

```diff
@@ -15,7 +15,7 @@ def create_app(self):
 
     def test_dataset_get(self):
         """Test case for dataset_get"""
-        expected_response = {'name': dataset_name}
+        expected_response = {'name': dataset_name, 'enable_search': False}
```

Comment:
Let's revert this change, we want to make sure things work if ui.json doesn't have this flag

Reply:
Got it